python/tank/bootstrap/configuration_writer.py

```diff
@@ -34,7 +34,7 @@ class ConfigurationWriter(object):
     Class used to write and update Toolkit configurations on disk.
     """
 
-    def __init__(self, path, sg):
+    def __init__(self, path, sg, is_unmanaged=True):
```

Comment:
so to followup on our discussion: let's drop the `is_unmanaged` part of the PR and not redefine the meaning of unmanaged, but keep the rest!